Files/UserControls/StatusBarControl.xaml

```diff
@@ -137,7 +144,7 @@
         <VisualStateManager.VisualStateGroups>
             <VisualStateGroup x:Name="Default">
                 <VisualState x:Name="NoOperations" />
-                <VisualState x:Name="InProgress">
+                <!--<VisualState x:Name="InProgress">
```

Comment:
I'd comment out (or plainly remove) also lines 135 to 143 (`<Interactivity:Interaction.Behaviors>` ...), since it refers to this VisualState (probably not an issue but hey)

Reply:
Good idea, thanks.